ext/xml/xml.c

```diff
@@ -1178,16 +1178,14 @@ static bool php_xml_check_string_method_arg(
 		&& parser_to_check->fcc_field.object == parser_to_check->object \
 		&& parser_to_check->fcc_field.calling_scope == NULL \
 	) { \
-		zend_string *method_name = zend_string_copy(parser_to_check->fcc_field.function_handler->common.function_name); \
+		zend_string *method_name = parser_to_check->fcc_field.function_handler->common.function_name; \
 		zend_fcc_dtor(&parser_to_check->fcc_field); \
```

Comment:
Have you checked with a trampoline? As in that case the function name could be released via `zend_fcc_dtor()` no?

Reply:
I think you're right; at least when there are NUL bytes in the function name, `zend_get_call_trampoline_func` makes a new string. I'll revert this. Nice find.